client/a8c-for-agencies/controller.tsx

```diff
@@ -70,6 +72,18 @@ export const requireClientAccessContext: Callback = ( context, next ) => {
 	);
 };
 
+export const requireLegacyClientBillingContext: Callback = ( context, next ) => {
+	const state = context.store.getState();
+	const clientBillingType = getClientBillingType( state );
+
+	if ( clientBillingType === 'legacy' ) {
```

Comment:
I wonder if we should flip this to instead check if it's NOT `billingdragon` with
`clientBillingType !== 'billingdragon'`

That makes it a clearer fallback where it's either `billingdragon` or not. 🤔 

I don't feel strongly about this though and could go either way. What do you think?

Reply:
Hi Travis, since `requireLegacyClientBillingContext` is used to assess legacy billing, it makes sense to check for the 'legacy' value here. I decided to leave it as is.
 
However, not exactly related, but I've made some improvements in the logic, so we test for `userBillingType !== 'legacy' and have the fallback go to `next()`. It is more natural to read the flow like this.